mathics/format/json.py

```diff
@@ -116,7 +116,7 @@ def cylinder_3d_box(self):
         [self.points],
         "cylinder",
         face_color,
-        {"faceColor": face_color, "radius": self.radius},
```

Comment:
Great! One less thing for the translation layer.

Reply:
But please check the other places in this file,  json.py, to make sure they all should be changed too.

Again, just add a PR on top of this.